Validation/RecoJets/plugins/JetTester.h

```diff
@@ -70,140 +73,160 @@ class JetTester : public DQMEDAnalyzer {
   edm::EDGetTokenT<pat::JetCollection> patJetsToken_;
   edm::EDGetTokenT<reco::JetCorrector> jetCorrectorToken_;
 
+  bool mInEtaBin(reco::Jet jet, double etaMin, double etaMax) {
+    return ((std::abs(jet.eta()) >= etaMin) && (std::abs(jet.eta()) < etaMax));
+  }
+
+  static const int ptSize = 10;
+  static constexpr std::array<double, ptSize + 1> ptBins_ = {
+      {20., 30., 40., 100., 200., 300., 600., 2000., 5000., 6500., 1e6}};
+  double minJetPt;
+
+  static constexpr std::array<std::tuple<std::string, std::string, double, double>, 3> etaInfo = {{
+      std::make_tuple("B", "0<|#eta|<1.5", 0.0, 1.5),     // barrel
+      std::make_tuple("E", "1.5#leq|#eta|<3", 1.5, 3.0),  // endcap
+      std::make_tuple("F", "3#leq|#eta|<6", 3.0, 6.0),    // forward
+  }};
```

Comment:
This code seems to have broken UBSAN build, see https://github.com/cms-sw/cmssw/issues/48802